src/open_clip/model.py

```diff
@@ -255,6 +255,9 @@ def load_checkpoint(
     resize_pos_embed(state_dict, model)
     resize_text_pos_embed(state_dict, model)
 
+    state_dict = {key: value for key, value in state_dict.items() if 'rope' not in key}
```

Comment:
Shall I make these changes specific to model architecture using an if statement? In general these lines will change weights only on state dicts coming from an EVA architecture, (e.g. 'rope' and 'visual.pos_embed' strings are only found on EVA image towers).